src/main/bash/commands/kobman-install.sh

```diff
@@ -1,102 +1,50 @@
 #!/usr/bin/env bash
 
+confirmed_environment=""
```

Comment:
what does confirmed mean? does it Kobman environment  that is current , available on host or available on remote repo?